jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh

```diff
@@ -88,6 +88,20 @@ elif [ "${BUILD_MODE}" == "release" ] ; then
     SPEC_VERSION_COUNT=6
 fi
 
+echo
+echo "=========="
+echo "Setup OIT stuff"
+echo "=========="
+
+
+pushd ${BUILDPATH}
```

Comment:
no reason to push & pop that I see.

Reply:
Fair... was just copy pasted from OSE. Will remove.